src/main/java/org/avarion/graves/listener/PlayerDeathListener.java

```diff
@@ -21,7 +21,7 @@ public PlayerDeathListener(Graves plugin) {
         this.plugin = plugin;
     }
 
-    @EventHandler(priority = EventPriority.LOWEST)
+    @EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
```

Comment:
I don't think we should process cancelled events? There is a reason for that.
I do agree with the `EventHandler` priority being very strange (upside down)...

Would you mind adding this as a config option?

I'd imagine something like:
*config file*
```yaml
event-priority: "HIGHEST" # Options: LOWEST, LOW, NORMAL, HIGH, HIGHEST, MONITOR
```
*code*
```java
[..]
        EventPriority priority;
        try {
            priority = EventPriority.valueOf(priorityStr.toUpperCase());
        } catch (IllegalArgumentException e) {
            getLogger().warning("Invalid event priority in config: " + priorityStr + ". Defaulting to HIGHEST.");
            priority = EventPriority.HIGHEST;
        }
[..]
        getServer().getPluginManager().registerEvents(new PlayerDeathListener(this), this, priority);
[..]
@EventHandler()
```

WDYT?
```

Reply:
I assumed that this was the default?

Seems I was mistaken:
```
    boolean ignoreCancelled() default false;
```

Waiting for your update :)